x/pool-incentives/keeper/grpc_query_test.go

```diff
@@ -170,81 +156,53 @@ func (suite *KeeperTestSuite) TestLockableDurations() {
 }
 
 func (suite *KeeperTestSuite) TestIncentivizedPools() {
-	suite.SetupTest()
-
-	keeper := suite.App.PoolIncentivesKeeper
-	ammKeeper := suite.App.GAMMKeeper
-	queryClient := suite.queryClient
-	var poolId uint64
-
-	gaugePerpetualId, err := suite.App.IncentivesKeeper.CreateGauge(
-		suite.Ctx, isPerpetual, sdk.AccAddress{}, sdk.Coins{}, lockuptypes.QueryCondition{
-			LockQueryType: lockuptypes.ByDuration,
-			Denom:         "stake",
-			Duration:      time.Hour,
-		}, time.Now(), 1)
-	suite.Require().NoError(err)
-
-	gaugeNonPerpetualId, err := suite.App.IncentivesKeeper.CreateGauge(
-		suite.Ctx, notPerpetual, sdk.AccAddress{}, sdk.Coins{}, lockuptypes.QueryCondition{
-			LockQueryType: lockuptypes.ByDuration,
-			Denom:         "stake",
-			Duration:      time.Hour,
-		}, time.Now(), 1)
-
-	suite.Require().NoError(err)
 
 	for _, tc := range []struct {
 		desc                 string
 		poolCreated          bool
-		distRecords          []types.DistrRecord
-		expectedErr          bool
+		weights              []sdk.Int
+		perpetual		     bool
+		nonPerpetual		 bool
```

Comment:
if perpetual is false that means nonPerpetual is automatically true right?

Reply:
Ahh just optional variables to test cases with additional Perpetual or NonPerpetual gauges